src/docs/sections/services.hbs

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Services & Trustpilot Sections" }}
+</head>
+
+<body>
+  <section class="text-center bg-secondary-light">
+    <div class="container">
+      <ul class="list-unstyled row align-items-center justify-content-center pt-md-6 pb-md-6 pt-6 pb-0 pl-0 mx-0 mb-0 border-bottom border-secondary">
+        <li class="col-md-3 col-sm-6">
+          <i class="sni sni__fast-delivery d-inline-flex text-secondary h1 mb-0"></i>
+          <p class="title mb-md-2 mb-4">Fast worldwide<br/>
+          delivery</p>
+        </li>
+        <li class="col-md-3 col-sm-6">
+          <i class="sni sni__winner-award d-inline-flex text-secondary h1 mb-0"></i>
+          <p class="title mb-md-2 mb-4">Winner of<br/>
+            16 Beauty Awards
+          </p>
+        </li>
+        <li class="col-md-3 col-sm-6">
+          <i class="sni sni__guarantee d-inline-flex text-secondary h1 mb-0"></i>
+          <p class="title mb-md-2 mb-4">100% money back<br/>
+          guarantee</p>
+        </li>
+      </ul>
+    </div>
+    <div class="row">
+      <div class="col-12 pt-md-4 pb-md-4 pt-2 pb-2 col-md-8 mx-auto">
```

Comment:
we don't need row and col classes here (also you're not using them in a container), remove it and place the padding classes to the trustpilot DIV and move into the container above